lib/scrape.js

```diff
@@ -103,5 +103,5 @@ async function fetchOrgsWithData() {
     (a, b) => b.completed_task_instance_count - a.completed_task_instance_count
   )
 
-  fs.writeFileSync('out/data.json', JSON.stringify(data))
+  fs.writeFileSync('./out/data.json', JSON.stringify(data))
```

Comment:
Should be `../out` ?

Should be the same on `lib/generate.js` as well.

Reply:
Doesn't work, try building with the file path changes you suggested. It fails.

Reply:
Oh it does not work because you run it from the root folder.

You need to use `__dirname` to make sure it ran correctly regardless of the current directory.